src/main/java/frc/robot/Robot.java

```diff
@@ -36,7 +36,7 @@ public void robotInit() {
     // TODO: Put commands here
 
     usbCamera = CameraServer.startAutomaticCapture();
-    usbCamera.setResolution(320, 240);
+    usbCamera.setResolution(240, 320);
```

Comment:
This seem to break simulation in Windows, though it apparently doesn't complain on the pipeline Linux machine. 
Wrap with if (isReal()) {}